tests/Unit/SprykerEco/Zed/ArvatoRss/Business/Api/Mapper/RiskCheckRequestMapperTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Unit\SprykerEco\Zed\ArvatoRss\Business\Api\Mapper;
+
+use ArrayObject;
+use Generated\Shared\Transfer\ArvatoRssBillingCustomerTransfer;
+use Generated\Shared\Transfer\ArvatoRssCustomerAddressTransfer;
+use Generated\Shared\Transfer\ArvatoRssIdentificationRequestTransfer;
+use Generated\Shared\Transfer\ArvatoRssOrderItemTransfer;
+use Generated\Shared\Transfer\ArvatoRssOrderTransfer;
+use Generated\Shared\Transfer\ArvatoRssRiskCheckRequestTransfer;
+use Spryker\Shared\Config\Config;
+use Spryker\Shared\Kernel\Store;
+use SprykerEco\Shared\ArvatoRss\ArvatoRssConstants;
+use SprykerEco\Zed\ArvatoRss\Dependency\Facade\ArvatoRssToMoneyBridge;
+
+class RiskCheckRequestMapperTest extends AbstractMapperTest
+{
+
+    /**
+     * @const int INT_VALUE
+     */
+    const INT_VALUE = 12475;
+
+    /**
+     * @const double DECIMAL_VALUE
+     */
+    const DECIMAL_VALUE = 124.75;
+
+    /**
+     * @const int UNIT_COUNT
+     */
+    const UNIT_COUNT = 1;
+
+    /**
+     * @const int PRODUCT_GROUP_ID
+     */
+    const PRODUCT_GROUP_ID = 1;
+
+    /**
+     * @dataProvider userDataProvider
+     *
+     * @param \ArrayObject $data
+     *
+     * @return void
+     */
+    public function testMapQuoteToRequestTranfer(ArrayObject $data)
+    {
+        $quoteTranfer = $this->quoteHelper->createQuoteTransfer($data);
+        $mapper = $this->helper->createRequestMapper($this->createMoneyFacadeMock());
+
+        $expected = $this->getExpectedRequestTransfer($data)->toArray(true);
+
+        $actual = $mapper->mapQuoteToRequestTranfer($quoteTranfer)->toArray(true);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    /**
+     * @return array
+     */
+    public function userDataProvider()
+    {
+        return [
+            [
+                new ArrayObject(
+                    [
+                        'clientId' => Config::get(ArvatoRssConstants::ARVATORSS_CLIENTID),
```

Comment:
Are you sure these keys are not restricted ?
I think those should be used from one of consts files